notes/returned-mail.vbs

```diff
@@ -69,23 +69,22 @@ family_cash_case, mfip_case, dwp_case, adult_cash_case, ga_case, msa_case, grh_c
 verifications_requested, other_notes, due_date, POLI_TEMP_PACT_button, panel_number, panel_number_check, TIKL_note_text, _
 SNAP_POLI_TEMP_button, CASH_POLI_TEMP_button, one_source_button, err_msg, are_we_passworded_out, case_active, MAXIS_footer_year, MAXIS_footer_month, case_pending, case_rein, ma_case, _
 msp_case, emer_case, unknown_cash_pending, unknown_hc_pending, ga_status, msa_status, mfip_status, dwp_status, grh_status, snap_status, ma_status, msp_status, msp_type, emer_status, _
-emer_type, case_status, list_active_programs, list_pending_programs, mail_city, ADHI_button, HSR_manual_button, new_addr_line_one, new_addr_line_two, new_addr_city, new_addr_state, _
+emer_type, case_status, list_active_programs, list_pending_programs, ADHI_button, HSR_manual_button, new_addr_line_one, new_addr_line_two, new_addr_city, new_addr_state, _
 new_addr_zip, new_addr_street_full, begining_of_footer_month, resi_street_full, county_code, date_verifications_requested, error_message, mets_addr_correspondence, open_cash1, open_cash2, _
-pact_pop_up, closing_message, end_msg
-
+pact_pop_up, closing_message, end_msg, received_error_confirmation
 
 EMConnect ""                                        'Connecting to BlueZone
 CALL MAXIS_case_number_finder(MAXIS_case_number)    'Grabbing the CASE Number
-call Check_for_MAXIS(false)                         'Ensuring we are not passworded out
-back_to_self                                        'added to ensure we have the time to update and send the case in the background
+CALL Check_for_MAXIS(false)                         'Ensuring we are not passworded out
+CALL back_to_self                                        'added to ensure we have the time to update and send the case in the background
 
 '-------------------------------------------------------------------------------------------------DIALOG
 Dialog1 = "" 'Running the initial dialog
-BeginDialog Dialog1, 0, 0, 221, 225, "RETURNED MAIL PROCESSING"
+BeginDialog Dialog1, 0, 0, 221, 225, "RETURNED MAIL"
   EditBox 75, 65, 50, 15, MAXIS_case_number
   EditBox 75, 85, 50, 15, date_received
   EditBox 75, 105, 50, 15, METS_case_number
-  DropListBox 105, 125, 110, 15, "Select:"+chr(9)+"forwarding address provided"+chr(9)+"no forwarding address provided"+chr(9)+"no response received", ADDR_actions
+  DropListBox 90, 125, 125, 15, "Select:"+chr(9)+"forwarding address provided"+chr(9)+"no forwarding address provided"+chr(9)+"no response received"+chr(9)+"address confirmed - received in error", ADDR_actions
```

Comment:
Initial dialog suggestion - change text 'mail has been returned' to 'Mail status'

'Address confirmed - received in error' in the drop list verbiage is cut off

Reply:
Changed  'mail has been returned' to "Returned Mail:" and adjusted the size of drop list